v0.5.0 closes #34 (#35)
turbot/steampipe-plugin-digitalocean: CHANGELOG.md

## v0.5.0 [2021-08-05]

_What's new?_

- New tables added
  - [digitalocean_alert_policy](https://hub.steampipe.io/plugins/turbot/digitalocean/tables/digitalocean_alert_policy) ([#33](https://github.com/turbot/steampipe-plugin-digitalocean/pull/33))

_Enhancements_

>   - Updated: Recompiled plugin with [Steampipe-plugin-sdk v1.4.1](https://github.com/turbot/steampipe-plugin-sdk/blob/main/CHANGELOG.md#v131--2021-07-15)([#22](https://github.com/turbot/steampipe-plugin-digitalocean/pull/22))

Reviewer: ```suggestion
  - Updated: Recompiled plugin with [steampipe-plugin-sdk v1.4.1](https://github.com/turbot/steampipe-plugin-sdk/blob/main/CHANGELOG.md#v131--2021-07-15)([#22](https://github.com/turbot/steampipe-plugin-digitalocean/pull/22))
```

_Bug fixes_
  - Fixed: Example query updated in `digitalocean_volume` table ([#28](https://github.com/turbot/steampipe-plugin-digitalocean/pull/28))

>   - Fixed: Next backup window time no longer cause queries to fail in the `digitalocean_droplet` table ([#24](https://github.com/turbot/steampipe-plugin-digitalocean/pull/24))

Reviewer: ```suggestion
  - Fixed: Querying data for columns `x` and `y` no longer causes queries to fail in the `digitalocean_droplet` table ([#24](https://github.com/turbot/steampipe-plugin-digitalocean/pull/24))
```
